$OpenBSD$

Fix both warning :
  warning: implicit declaration of function 'getpid'
  warning: implicit declaration of function 'usleep'

--- src/lib/ecore/ecore_thread.c.orig	Wed Oct 26 03:37:30 2011
+++ src/lib/ecore/ecore_thread.c	Wed Oct 26 03:38:09 2011
@@ -5,6 +5,9 @@
 #include <sys/time.h>
 #include <assert.h>
 
+#include <sys/types.h>
+#include <unistd.h>
+
 #ifdef HAVE_EVIL
 # include <Evil.h>
 #endif
